### ISSUE TITLE
[LOW PRIORITY!] dedicated Follow Hashtag Icon

### ISSUE THREAD

**Issue by OddSobriquet** [mastodon/mastodon#21410]
### Pitch

While the usage of the regular Follow-Icon does the job I think a dedicated Follow Hashtag Button Icon would improve the UX.

### Motivation

The Follow this Person Icon had me a little confused the first time I encountered it when trying to follow a hashtag. A dedicated Icon would improve UX by clarifying one is about to subscribe to a hashtag.

**Comment by Cassolotl** [comment]
Yeah, it looks like a profile button so it did add confusion for me too!
